samples/nrf9160/location/prj.conf

```diff
@@ -56,5 +56,7 @@ CONFIG_DATE_TIME=y
 # Modem JWT
 CONFIG_MODEM_JWT=y
 
-# Multicell location service selection
-CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+# Location service selection
+#CONFIG_LOCATION_SERVICE_NRF_CLOUD=y
+#CONFIG_LOCATION_SERVICE_HERE=y
+#CONFIG_LOCATION_SERVICE_HERE_API_KEY="api key"
```

Comment:
set the API key to a blank string so the BUILD_ASSERT will catch it if not set by the user?